doc/proxy-clients.rst

```diff
@@ -184,6 +184,42 @@ dispatcher as explained above and pass it to the Symfony client::
 
     $symfony = new Symfony($httpDispatcher);
 
+
+The `HttpDispatcher` sends real HTTP requests using any instance of `HttpAsyncClient`
+available in your application (see HTTPlug_ for more information). If you
+have a single node Symfony application setup you might want to directly route
```

Comment:
s/node/server/ 

application setup, you can send these requests directly to the cache kernel inside the same PHP process instead of sending actual HTTP requests over the network.